[INFO   ] Logger: Record log in /storage/emulated/0/flappybird/.kivy/logs/kivy_22-12-12_18.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  6 2021, 01:34:26) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Audio: Providers: audio_sdl2 (audio_android, audio_ffpyplayer ignored)
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/flappybird/main.py", line 261, in <module>
[WARNING] stderr:     main().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 949, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 918, in _run_prepare
[WARNING] stderr:     self.load_kv(filename=self.kv_file)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 691, in load_kv
[WARNING] stderr:     root = Builder.load_file(rfilename)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 306, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 373, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 402, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 511, in parse
[WARNING] stderr:     objects, remaining_lines = self.parse_level(0, lines)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 614, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 614, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 614, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 591, in parse_level
[WARNING] stderr:     raise ParserException(self, ln, 'Invalid class name')
[WARNING] stderr: kivy.lang.parser.ParserException: Parser: File "/storage/emulated/0/flappybird/main.kv", line 163:
[WARNING] stderr: ...
[WARNING] stderr:     161:                    pos:self.pos
[WARNING] stderr:     162:                    
[WARNING] stderr: >>  163:            canvas.after:
[WARNING] stderr:     164:                #PopMatrix:
[WARNING] stderr:     165:                
[WARNING] stderr: ...
[WARNING] stderr: Invalid class name
